src/components/layout/header/DrawerMenu/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { MouseEventHandler, RefObject } from 'react';
+
+import classNames from 'classnames/bind';
+
+import { GAME_LIST, SVGS } from '@/constants';
+import { formatGameToLink } from '@/utils';
+
+import styles from './DrawerMenu.module.scss';
+
+const cx = classNames.bind(styles);
+const { url, alt } = SVGS.close;
+
+type DrawerMenuProps = {
+  drawerMenuRef: RefObject<HTMLDivElement>;
+  handleToggleDrawerMenu: MouseEventHandler<HTMLButtonElement>;
+};
+
+export const DrawerMenu = ({ drawerMenuRef, handleToggleDrawerMenu }: DrawerMenuProps) => {
+  return (
+    <div className={cx('drawer-menu')} ref={drawerMenuRef}>
+      <button className={cx('drawer-menu-close-button')} onClick={handleToggleDrawerMenu}>
+        <Image src={url} alt={alt} width={24} height={24} />
+      </button>
+      <nav>
+        <ul className={cx('drawer-menu-game-list')}>
+          {GAME_LIST.map((game, index) => (
+            <li className={cx('drawer-menu-game')} key={`game-${index}`}>
+              <Link className={cx('drawer-menu-game-link')} href={formatGameToLink(game)}>
```

Comment:
```suggestion
            <li className={cx('drawer-menu-game-list-item')} key={`game-${index}`}>
              <Link className={cx('drawer-menu-game-item-link')} href={formatGameToLink(game)}>
```

이런 식으로 nesting 문법을 쓰면 좋을 것 같습니다.
이렇게 사용하면 css에서도
```
 drawer-menu-game-list {
 &-item {
   &-link { ...}
 }
}
```
이렇게 쓸 수 있으니 가독성도 더 좋아질 것 같습니다. 지금 css 파일을 보면 네스팅 문법을 어느 곳에선 적용하고 어느 곳에선 적용하지 않아서 헷갈리는 부분이 있습니다~

Reply:
반영했습니다!